### DISCUSSION TITLE
Scoping review with ASReview

### DISCUSSION THREAD

**Discussion by vissereva** [asreview/asreview#908] [Q&A]
Dear teammembers of ASReview,

For my PhD I’m doing a scoping review and I was wondering if your software can be used for that purpose too. Since the first step of a scoping review and a systematic review is similar in terms of screening, I thought that might be possible. Is that right? And is there any experience with using ASReview for screening for a scoping review? 
Thanks in advance. 

Kind Regards,
Eva Visser

**Comment by jaohls** [comment]
Hi,

Not an ASReview team member, but our research group has chosen to use ASReview for a scoping review. As you write yourself, the screening stage should be more or less identical to systematic reviews, and saving work hours on the screening could allow for a broader initial search strategy.

So far we've only done piloting, but will commence screening soon. We're working on 3 different topics/searches, with 2 researchers doing the screening for each topic.

Hope this helps - might have further insights to share once we get started properly.


EDIT: spelling.

**Reply by jaohls** [reply]
... our protocol can be found here: https://osf.io/dv3uy (full .pdf under "files")

**Comment by Rensvandeschoot** [comment]
See also these publications discussing a systematic scoping review: 

- [Implementation and dissemination of home and community-based interventions for informal caregivers of people living with dementia: a systematic scoping review protocol](https://bmjopen.bmj.com/content/12/1/e052324)

- [Clinical applicability of artificial intelligence for patients with an inherited heart disease: a scoping review](https://doi.org/10.1016/j.tcm.2022.01.011)
